plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-interval.php

```diff
@@ -612,6 +612,20 @@ public function test_intervals_between() {
 					'year'    => 4,
 				),
 			),
+			// Test change of DST times.
+			array(
+				'start'      => '2021-10-01T00:00:00',
+				'end'        => '2021-10-31T23:59:59',
+				'week_start' => 1,
+				'intervals'  => array(
+					'hour'    => 31 * 24,
+					'day'     => 31,
+					'week'    => 5,
+					'month'   => 1,
+					'quarter' => 1,
+					'year'    => 1,
+				),
+			),
```

Comment:
Should we be explicitly setting the timezone to a DST adhering timezone for this test? Given it might not be using that as the timezone?

Reply:
Good idea.  Updated to set a specific timezone in https://github.com/woocommerce/woocommerce/pull/33036/commits/15decc9301f9bbaa75b1e9b374b7f818c006db06.